tests/integration/linux_bridge_test.py

```diff
@@ -269,6 +277,15 @@ def test_replace_port_on_linux_bridge(port0_vlan101, port1_up):
         )
 
 
+def test_linux_bridge_over_bond_over_slave(bond0):
```

Comment:
Could you please confirm that this test will fail without this fix?

Reply:
In this new updated version, it does - it is not deterministic; it fails around 50% of the times.

I need to think how to update the commit msg of this patch to reflect that.